.circleci/config.yml

```
@@ -104,28 +103,54 @@ jobs:
             - /tmp/docker
 
   get_data:
-    machine:
-      # Ubuntu 14.04 with Docker 17.10.0-ce
-      image: circleci/classic:201711-01
-    working_directory: /home/circleci/data
+    docker:
+      - image: continuumio/miniconda3:4.9.2-alpine
+    working_directory: /tmp/data
+    environment:
+      - TEMPLATEFLOW_HOME: /tmp/templateflow
     steps:
       - restore_cache:
           keys:
-            - data-v0-{{ .Branch }}-{{ .Revision }}
-            - data-v0-{{ .Branch }}
-            - data-v0-
+            - env-v0-{{ .Branch }}-
+            - env-v0-master-
+            - env-v0-
+      - run:
+          name: Setup git-annex & DataLad
+          command: |
+            conda install -y -c anaconda -c conda-forge git-annex datalad
```

is there a need to add the anaconda channel if both of those are on conda-forge?
```suggestion
            conda install -y -c conda-forge git-annex datalad
```

---

AFAIK git-annex is not on conda-forge

---

FWIW: https://github.com/conda-forge/git-annex-feedstock